assets/queries/cloudFormation/gamelift_fleet_ec2_inbound_permissions_with_port_range/query.rego

```diff
@@ -16,3 +18,60 @@ CxPolicy[result] {
 		"keyActualValue": sprintf("Resources.%s.Properties.EC2InboundPermissions[%d].FromPort is not equal to Resources.%s.Properties.EC2InboundPermissions[%d].ToPort", [name, index, name, index]),
 	}
 }
+
+CxPolicy[result] {
```

Comment:
Could use `compareRuleNumber` instead of replicating same rule?

Reply:
Couldn't use "compareRuleNumber" as it is specific to the other query and resource, but simplified the query using the other change you suggested